Automattic/wp-calypso
# Bump Newspack Posts Block to v1.2.1

#### Changes proposed in this Pull Request
* Bump to v1.2.1
* Fix a PHP notice I just saw in wp-admin (visible when rendering a non-post wp-admin page)

#### Testing instructions
* Sync to dotcom and make sure everything is working properly on a sandboxed site.
* Test that the author URL is fixed on the front end (https://github.com/Automattic/newspack-blocks/pull/407)
